gateway-messages/src/mgs_to_sp.rs

```diff
@@ -204,6 +204,11 @@ pub enum MgsRequest {
     ComponentWatchdogSupported {
         component: SpComponent,
     },
+
+    /// Read RoT boot state at the highest version not to exceed specified version.
```

Comment:
Just a question about the wording here: I understand this to mean that if MGS requests `version: 5` and the RoT supports up to `version: 8`, the RoT should send back a version 5 state struct. What if MGS requests `version: 5` and the RoT only supports up to `version: 3`?

Reply:
Rot returns version 3 then. It doesn't know anything higher.